test/test_python_dispatch.py

```diff
@@ -376,7 +376,7 @@ def __new__(cls, elem, *args, **kwargs):
 
             @classmethod
             def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
-                if func.__name__ == "clone":
+                if func.overloadpacket.__name__ == "clone":
```

Comment:
What is `__name__` on the overload func now?

Reply:
`op_name.overload_name`